tests/test_filters.py

```diff
@@ -172,11 +172,11 @@ class SearchListView(generics.ListAPIView):
             search_fields = ('$title', '$text')
 
         view = SearchListView.as_view()
-        request = factory.get('/', {'search': '^\\w{3}$'})
+        request = factory.get('/', {'search': 'r^\w{3}$'})
         response = view(request)
         assert len(response.data) == 10
 
-        request = factory.get('/', {'search': '^\\w{3}$', 'title_only': 'true'})
+        request = factory.get('/', {'search': 'r^\w{3}$', 'title_only': 'true'})
```

Comment:
The `r` needs to be on the outside of the quote 😄

Reply:
sorry!!

Reply:
done

Reply:
👌